gatsbyjs/gatsby
# docs: v3 Tutorial

## Description

Update the Tutorial to use the latest and greatest Gatsby v3 features!

This also updates the Tutorial to make all sections build on the same project (as opposed to creating a separate Gatsby site in each part).

### Documentation

* `/docs/tutorial/part-0`
* `/docs/tutorial/part-1`
* `/docs/tutorial/part-2`
* ~~`/docs/tutorial/part-3`~~

The new Tutorial will be released in chunks, to balance getting new content to users quickly with taking the time needed for quality writing. This first release will include Parts 0-2. Parts 3-7 will come in subsequent PRs.

To test locally:

1. Check out mansion branch `docs/v3-tutorial`
2. Check out gatsby branch `v3-tutorial`
3. From `mansion/cloud/gatsbyjs.com`, run `yarn run develop:docs`

## To Do

- [x] Write Part 0: Set Up Your Development Environment.
- [x] Write Part 1: Create and Deploy Your First Gatsby Site.
- [x] Write Part 2: Use and Style React Components.
- [ ] ~~Write Part 3: Add Features with Plugins.~~
- [x] Update callout boxes to use a component with `<aside>` instead of misusing the blockquote Markdown element. (Used the `Announcement` component from `gatsby-interface` instead.
- [x] Part 0: Update the Node.js install instructions to use a clearer visual interface for the collapsible OS-specific sections.
- [ ] ~~Delete all the files for the v2 Tutorial. (Folks looking for the old Tutorial can access it at https://v2.gatsbyjs.com/docs/tutorial/)~~ Don't actually do this yet, to make it easier to coordinate the release of this PR with the one in mansion.
